pom.xml

```diff
@@ -365,6 +365,26 @@
         <configuration>
           <showLogs>true</showLogs>
           <images>
+            <image>
+              <name>joeyjackson/kafka:${project.version}</name>
+              <build>
+                <dockerFile>${project.basedir}/src/main/docker/kafka/Dockerfile</dockerFile>
```

Comment:
Looks good. Can you also move the MAD `Dockerfile` say into `.../src/main/docker/mad/Dockerfile`? You will need to update the reference in the assembly xml: `src/main/assembly/docker.xml`